src/resultsdisassemblypage.cpp

```diff
@@ -199,3 +201,46 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
         m_objdump = QLatin1String("aarch64-linux-gnu-objdump");
     }
 }
+
+void ResultsDisassemblyPage::resetCallStack()
+{
+    m_callStack.clear();
+}
+
+void calcFunction(QString asmLineCall, QString* offset, QString* symName)
+{
+    QStringList sym = asmLineCall.trimmed().split(QLatin1Char('<'));
+    *offset = sym[0].trimmed();
+    *symName = sym[1].replace(QLatin1Char('>'), QLatin1String(""));
+}
+
+void ResultsDisassemblyPage::jumpToAsmCallee(QModelIndex index)
+{
+    QStandardItem* asmItem = model->takeItem(index.row(), 0);
+    QString opCodeCall = m_arch.startsWith(QLatin1String("arm")) ? QLatin1String("bl") : QLatin1String("callq");
```

Comment:
please parse the arch once instead of doing that repeatedly here, also use auto on the left  hand side (i.e. make the result a QLatin1String)